source/implementations/f7/Meadow.F7/Devices/F7CellNetworkAdapter.cs

```diff
@@ -200,11 +210,136 @@ public string AtCmdsOutput
     }
 
     /// <summary>
-    /// Returns the list of cell networks found, including its operator code, if the device is in scanning mode, otherwise <b>null</b>
+    /// Returns the list of cell networks found, including its operator code, if the device is in scanning mode; otherwise, an empty array
     /// </summary>
-    public CellNetwork[] Scan()
+    public CellNetwork[] OfflineNetworkScan()
```

Comment:
Do you think that we should keep the Scan name for this method @alexischr ? If we decide to change, it might break some customer applications, and we'll need to adapt the docs and the cell basics sample. However, if we decide not to change, I think it might confuse the users, since now we'll have two scans, the online and the offline one.

Reply:
We're very early on, and very few customers are using the API.  It's best to make breaking changes now where it affects only a few people.